app/backend/src/tests/test_fixtures.py

```diff
@@ -277,13 +277,17 @@ def real_jail_session(token):
             server.stop(None).wait()
 
 
+def fake_channel(token):
+    user_id, jailed = Auth().get_session_for_token(token)
+    return FakeChannel(user_id=user_id)
+
+
```

Comment:
Thanks, this is good. Could you also change the last remaining `api_session` to use `fake_channel`?